### DISCUSSION TITLE
Feature Request: Add a feature to disable process/thread messages

### DISCUSSION THREAD

**Discussion by billgan1024** [microsoft/vscode-cmake-tools#3456] [Ideas]
Add a feature to disable process/thread messages in debug output like Visual Studio, to remove these messages:
 
![image](https://github.com/microsoft/vscode-cmake-tools/assets/17630457/f69254ab-3f60-477f-ba4d-b25d497d2af3)

 
 
 Currently a workaround is to use the c/c++ launch.json, as highlighted in the C/C++ extension repo:
 https://github.com/microsoft/vscode-cpptools/issues/5861

**Comment by gcampbell-msft** [comment]
@billgan1024 Thanks for the feedback and apologies for the long delay in response. I'm going to close the discussion due to age, but we'd be more than willing to investigate this request if you create a new issue for it that we can then track. Thanks!
